README.md

```diff
@@ -106,3 +106,11 @@ Getting Started on AWS
     --root-drive=hello-rootfs.ext4 \
     --kernel-opts="console=ttyS0 noapic reboot=k panic=1 pci=off nomodules rw"
   ```
+
+Testing
+---
+By default the tests require the firectl binary to be built and a kernel image
+to be present. The integration tests look for the binary and kernel image in
+the root directory. By default it will look for vmlinux kernel image. This can
+be overwritten by setting `KERNELIMAGE` to the desired path. To disable these
+tests simply set `SKIP_INTEG_TEST`.
```

Comment:
Because there's a distinction between an **unset** environment variable and a variable that's set to an **empty value**, it's actually important to clarify that `SKIP_INTEG_TEST` needs to be set to an actual value with a nonzero length. Rather than being so wordy, though, I'd just document that the user should set `SKIP_INTEG_TEST=1`.

Also, just to make it clear to the reader, please include a note indicating that these are environment variables.